### PR TITLE
Update URLs in comments to reflect migration to github.com

### PR DESCRIPTION
**Pull request by ChrisHines**

### CODE REVIEW

File: mssql_test.go [hunk 1]
```diff
@@ -219,7 +219,7 @@ func is2008OrLater(db *sql.DB) bool {
 }
 
 func exec(t *testing.T, db *sql.DB, query string) {
-	// TODO(brainman): make sure http://code.google.com/p/go/issues/detail?id=3678 is fixed
+	// TODO(brainman): make sure https://github.com/golang/go/issues/3678 is fixed
```
Comment (ChrisHines) [inline mssql_test.go]: This issue is marked as fixed. Anything to change here?

**Comment by alexbrainman** [comment]
LGTM. Thank you very much.

Would you mind sending new PR to add yourself to CONTRIBUTORS? Thank you.

Alex

**Comment by ChrisHines** [comment]
Sure. Question: I sometimes have different email addresses in my Git commits depending on various things. Does it matter if the email I put in CONTRIBUTORS matches with the one on my commits? CONTRIBUTORS mentions a second email for use with Rietveld, but that is obviously outdated.

**Comment by alexbrainman** [comment]
Feel free to put multiple email addresses on your line in CONTRIBUTORS. See Go project CONTRIBUTORS file. Thank you.

Alex

**Pull request merged**